redbox/llm/spotlight/spotlight.py

```diff
@@ -9,29 +11,33 @@
 
 # region ===== TASKS =====
 
+CREATOR_USER_UUID = UUID("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa")
```

Comment:
this is a bit nasty but a price worth paying to ensure that data is always saved with a `creator_user_id`?

Reply:
Do we need to create 2 persistable models? One with and one without user_id?

Reply:
https://technologyprogramme.atlassian.net/browse/REDBOX-208